backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentDatabaseHandler.java

```diff
@@ -273,6 +273,15 @@ private void autosetReleaseClearingState(Release releaseAfter, Release releaseBe
     ////////////////////////////
     // GET INDIVIDUAL OBJECTS //
     ////////////////////////////
+    
+    public void addSelectLogs(Component component, User user){
```

Comment:
Please check Indentation before { 
same for line 281

Reply:
Fixed it.